bot/exts/utilities/rfc.py

```diff
@@ -0,0 +1,65 @@
+import datetime
+import logging
+
+from discord import Embed
+from discord.ext import commands
+
+from bot.bot import Bot
+from bot.constants import Colours
+
+logger = logging.getLogger(__name__)
+
+BASE_URL = "https://datatracker.ietf.org/doc/rfc{query}/doc.json"
+
+
+class Rfc(commands.Cog):
+    """Retrieve RFCs from their ID."""
+
+    def __init__(self, bot: Bot):
+        self.bot = bot
+
+    @commands.cooldown(1, 10, commands.BucketType.user)
+    @commands.command()
+    async def rfc(self, ctx: commands.Context, query: int) -> None:
```

Comment:
```suggestion
    async def rfc(self, ctx: commands.Context, rfc_id: int) -> None:
```
This will keep it more explicit, instead of guessing what a query is, or reading down to understand that a query is the id of an rfc

The API offers to search by document name, rfc id, etc, but we're only allowing search by id (maybe it could be a good idea to search by name ?), that's why I made the suggestion.